module_utils/helpers.py

```diff
@@ -488,6 +530,51 @@ def set_twophase_options_from_params(control_console, params):
     set_twophase_options(control_console, twophase_options)
 
 
+def read_yaml_file(control_console, file_path):
+    return control_console.eval_res_err(READ_YAML_FILE_FUNC_BODY, file_path)
+
+
+def get_clusterwide_config(control_console, filter_system=True):
+    current_config, err = control_console.eval_res_err('''
+        return require('cartridge').config_get_readonly()
+    ''')
+    if err is not None:
+        return None, err
+    if current_config is None:
+        return None, "Cluster isn't bootstrapped yet"
+
+    filtered = {}
+    for key, value in (current_config or {}).items():
+        if not key.endswith('.yml') and not (filter_system and key in SYSTEM_CONFIG_SECTIONS):
+            filtered[key] = value
+    return filtered, None
+
+
+def patch_clusterwide_config(control_console, new_sections):
+    current_config, err = get_clusterwide_config(control_console)
+    if err is not None:
+        return None, err
+
+    patch = {}
+    for section_name, section in new_sections.items():
```

Comment:
What about checking all sections and show one error for all system sections?
Smth like 

```python
passed_system_sections = [
    name for name in new_sections if name in new_sections
]

if passed_system_sections:
    return None, "... %s" % ', '.join(passed_system_sections)
```

Reply:
Good idea!